pymaid/pymaid.py

```diff
@@ -1164,63 +1165,105 @@ def get_connectors ( skids, remote_instance = None, incoming_synapses = True, ou
             remote_instance = globals()['remote_instance']
         else:
             print('Please either pass a CATMAID instance or define globally as "remote_instance" ')
-            return    
+            return
 
     if type(skids) != type(list()):
         skids = [ skids ]
 
-    get_connectors_GET_data = { 'with_tags': 'false' }
+    get_connectors_GET_data = { 'with_tags': 'true' }
 
     cn_data = []
 
+    #Initialize tags dict
+    tags_dict = {}
+
     #There seems to be some cap regarding how many skids you can send to the server, so we have to chop it into pieces
     for a in range( 0, len(skids), 50 ):
         for i,s in enumerate(skids[ a:a+50] ):
-            tag = 'skeleton_ids[%i]' % i 
+            tag = 'skeleton_ids[%i]' % i
             get_connectors_GET_data[tag] = str( s )
 
         if incoming_synapses is True:
             get_connectors_GET_data['relation_type']='presynaptic_to'
             remote_get_connectors_url = remote_instance.get_connectors_url( project_id ) + '?%s' % urllib.parse.urlencode(get_connectors_GET_data)
             cn_data += [ e + ['presynaptic_to'] for e in remote_instance.fetch( remote_get_connectors_url )['links'] ]
 
+            #Add tags to tags dict
+            tags_dict = remote_instance.fetch( remote_get_connectors_url )['tags']

```

Comment:
Fetching the URL twice is very inefficient. I would suggest something like this:
```Python
data = remote_instance.fetch( remote_get_connectors_url )
cn_data += [ e + ['presynaptic_to'] for e in data['links'] ]
tags_dict = data
```